tests/test_hexary_trie.py

```diff
@@ -166,13 +168,20 @@ def unread_keys(self):
 def test_hexary_trie_saves_each_root():
```

Comment:
nitpick: This test now tests a couple different things.

Well, it seems like an oversight that it was previously only checking whether there were extra items at all! It's a nice addition that it also verifies that at least part of the older tries are accessible.

However `at_root` maybe deserves it's own test.